src/Avalonia.Themes.Default/TabControl.xaml

```diff
@@ -1,50 +1,64 @@
 <Styles xmlns="https://github.com/avaloniaui"  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
-  <Style Selector="TabControl">
-    <Setter Property="Template">
-      <ControlTemplate>
-        <Border Background="{TemplateBinding Background}"
-                BorderBrush="{TemplateBinding BorderBrush}"
-                BorderThickness="{TemplateBinding BorderThickness}">
-          <DockPanel>
-            <TabStrip Name="PART_TabStrip"
-                      MemberSelector="{x:Static TabControl.HeaderSelector}"
-                      Items="{TemplateBinding Items}"
-                      SelectedIndex="{TemplateBinding SelectedIndex, Mode=TwoWay}"/>
-            <Carousel Name="PART_Content"
-                      MemberSelector="{x:Static TabControl.ContentSelector}"
-                      Items="{TemplateBinding Items}"
-                      SelectedIndex="{TemplateBinding SelectedIndex}"
-                      PageTransition="{TemplateBinding PageTransition}"
-                      Grid.Row="1"/>
-          </DockPanel>
-        </Border>
-      </ControlTemplate>
-    </Setter>
-  </Style>
-  <Style Selector="TabControl[TabStripPlacement=Top] /template/ TabStrip">
-    <Setter Property="DockPanel.Dock" Value="Top"/>
-  </Style>
-  <Style Selector="TabControl[TabStripPlacement=Bottom] /template/ TabStrip">
-    <Setter Property="DockPanel.Dock" Value="Bottom"/>
-  </Style>
-  <Style Selector="TabControl[TabStripPlacement=Left] /template/ TabStrip">
-    <Setter Property="DockPanel.Dock" Value="Left"/>
-    <Setter Property="ItemsPanel">
-      <Setter.Value>
-        <ItemsPanelTemplate>
-          <StackPanel Orientation="Vertical"/>
-        </ItemsPanelTemplate>
-      </Setter.Value>
-    </Setter>
-  </Style>
-  <Style Selector="TabControl[TabStripPlacement=Right] /template/ TabStrip">
-    <Setter Property="DockPanel.Dock" Value="Right"/>
-    <Setter Property="ItemsPanel">
-      <Setter.Value>
-        <ItemsPanelTemplate>
-          <StackPanel Orientation="Vertical"/>
-        </ItemsPanelTemplate>
-      </Setter.Value>
-    </Setter>
-  </Style>
-</Styles>
\ No newline at end of file
+    <Style Selector="TabControl">
+        <Setter Property="Background" Value="{DynamicResource ThemeBackgroundBrush}"/>
+        <Setter Property="BorderBrush" Value="{DynamicResource ThemeBorderMidBrush}"/>
+        <Setter Property="BorderThickness" Value="{DynamicResource ThemeBorderThickness}"/>
```

Comment:
Do we want to have a border around the `TabControl` by default? IMO it looks a little strange and as far as I know, UWP (which the theme is based on) doesn't have it.

Reply:
Removing it is fine for me will do so.